### ISSUE TITLE
DEBUG level not taking effect in main framework

### ISSUE THREAD

**Issue by StevenCTimm** [HEPCloud/decisionengine#132]
I am testing the framework version as set in Master as built July 2.

    "logger": {
	"log_file": "/var/log/decisionengine/decision_engine_log",
	"max_file_size": 200000000,
	"max_backup_count": 6,
	"log_level": "DEBUG"
	},

de-client --print-engine-loglevel correctly shows DEBUG but there are no DEBUG messages in decision_engine_log, or decision_engine_log_debug.  (we are only seeing INFO).

**Comment by DmitryLitvintsev** [comment]
these re the only debug messages in the framework:

```
mac-131313:decisionengine dmitrylitvintsev$ find ./ -name '*.py' -exec egrep -H '\.debug' {} \; 
.//framework/taskmanager/TaskManager.py:        logging.getLogger().debug('data_block_put %s', data)
.//framework/taskmanager/TaskManager.py:            logging.getLogger().debug('Duplicated block {}'.format(data_block))
.//framework/taskmanager/TaskManager.py:        logging.getLogger().debug('run_transforms: data block %s', data_block)
.//framework/taskmanager/TaskManager.py:                    logging.getLogger().debug('transform returned %s', data)
.//framework/taskmanager/TaskManager.py:            logging.getLogger().debug('run logic engine %s %s',
.//framework/taskmanager/TaskManager.py:                logging.getLogger().debug('run publisher %s %s',
.//framework/modules/de_logger.py:    my_logger.debug("THIS IS DEBUG")
.//framework/modules/SourceProxy.py:                self.logger.debug('task manager %s', tm)
.//framework/modules/SourceProxy.py:                    self.logger.debug('data block %s', data_block)
.//framework/modules/SourceProxy.py:                        self.logger.debug("DATABLOCK %s", data_block)
.//framework/modules/SourceProxy.py:                            self.logger.debug("KEYERROR %s", ke)
mac-131313:decisionengine dmitrylitvintsev$ 
```

you don't see those?

**Comment by StevenCTimm** [comment]
So basically all of the DEBUG statements happen in the TaskManager or lower, which goes in the individual logs instead of decision_engine_log and decision_engine_log_debug

I see all the stuff of data_block_put, run_transforms, run_logic_engine, run_publisher.  But I see them 
in the logs from their respective task manager.
I don't see the ones from SourceProxy.py because the only invocation of that is currently malfunctioning.. see other incidents that are open.

**Comment by DmitryLitvintsev** [comment]
> So basically all of the DEBUG statements happen in the TaskManager or lower, which goes in the individual logs instead of decision_engine_log and decision_engine_log_debug
> 
> I see all the stuff of data_block_put, run_transforms, run_logic_engine, run_publisher. But I see them
> in the logs from their respective task manager.
> I don't see the ones from SourceProxy.py because the only invocation of that is currently malfunctioning.. see other incidents that are open.

yes. TaskManager (aka channel) (and all the modules handled by TaskManager) runs in separate process and all logging wherein goes to specific channel log file. 

Only DecisionEnginge itself logs (or more precisely - anything running in main thread) to decision_engine_log and decision_engine_log_debug. 

So this is a non-issue?

**Comment by StevenCTimm** [comment]
I think this is a non-issue, but the related issue that just because there is DEBUG in the main file does not force
all channels to DEBUG level ( issue #109 ) remains, and would like to keep this one open until we are sure we are agreed that it is doing what it is intended to do.

**Comment by StevenCTimm** [comment]
This is fixed in the release candidate 1.3.0rc-1

**Comment by StevenCTimm** [comment]
Fixed, closing
